Orange/canvas/application/errorreporting.py

```diff
@@ -38,6 +38,20 @@ class OWWidget: pass
 log = logging.getLogger()
 
 
+def get_developer_id():
+    """Check if developer provided his name for Sentry reports."""
+    try:
+        from Orange.misc.environ import data_dir_base
+        fname = os.path.join(data_dir_base(), 'Orange', 'developer_id.txt')
```

Comment:
Btw, ORANGE_DEPRECATIONS_ERROR causes tests to fail locally if it is not set, so I do not think it is a good idea. Same goes for ORANGE_DEBUG, I would prefer to have a runtime argument (that is listed when canvas is run with --help)